overlord/fdestate/backend/reseal.go

```diff
@@ -338,6 +344,7 @@ func doReseal(manager FDEStateManager, rootdir string, hintExpectFDEHook bool, i
 				if !errors.Is(err, errNoPCRProfileCalculated) {
 					return err
 				}
+				newParameters.unset("recover", container.ContainerRole())
```

Comment:
```suggestion
                // no reseal required, don't overwrite current parameters to avoid missing PCR profile
				newParameters.unset("recover", container.ContainerRole())
```